tests/client/amo/components/TestAddonDetail.js

```diff
@@ -324,4 +325,12 @@ describe('AddonDetail', () => {
 
     assert.ok(rootNode.querySelector('.AddonMoreInfo-contents'));
   });
+
+  it('renders meta data', () => {
+    const root = render({ addon: fakeAddon });
```

Comment:
I totally missed that. I still prefer an explicit value but good to see that's covered.